zinc/src/sbt-test/source-dependencies/trait-trait-211/mirtest/EvaluatorSpecs.scala

```diff
@@ -0,0 +1,10 @@
+package xx
+
+import gg.table._
+
+trait EvaluatorSpecification extends EvaluatorTestSupport { self =>
+}
+
+trait EvaluatorTestSupport extends SliceTransforms { outer =>
```

Comment:
Why is the self annotation required here?

Reply:
I can remove that.